ansible/roles/th-web-connectivity/templates/if-nok-restart.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if curl -m 5 -s {{ web_connectivity_ipv4 }}:{{ web_connectivity_port }}/status | grep '{"status": "ok"}' > /dev/null
```

Comment:
do we want to add `-f` such that non `200` status codes are interpreted as bad?

Reply:
I suppose we could do that, however the thing that matters the most is the fact that the response code matches the grep parameter.

Moreover the only case we care about in reality is the one in which curl times out, which is what happens when it enters the failed OOM state.

Reply:
ok